### PR TITLE
FAQ Cause 2001

### PR DESCRIPTION
**Pull request by dsarkar**
Closes https://github.com/corona-warn-app/cwa-website/issues/782

---
Internal tracking ID: [EXPOSUREAPP-2009](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-2009)

### CODE REVIEW

**Comment by dsarkar** [comment]
EN

image

File: src/data/faq.json [hunk 1]
```diff
@@ -34,7 +34,19 @@
             {
                 "title": "Common Issues",
                 "id": "most_asked",
-                "accordion": [
+                "accordion": [
+                    {
+                        "title": "Cause: 2001, trying to e establish a secure connection to the server",
```
Comment (MikeMcC399) [inline src/data/faq.json:39]: trying to e establish
=> trying to establish

There is an extra "e" in the title.

Comment (dsarkar) [reply src/data/faq.json:39]: ok

**Comment by dsarkar** [comment]
DE

image

**Comment by dsarkar** [comment]
EN 

image

**Comment by dsarkar** [comment]
@MikeMcC399  Thanks.

**Comment by MikeMcC399** [comment]
@dsarkar 
The error in the title is in the merged German version.
Again, too quick for me to catch before merging 🙁 .

"URSACHE: 2001, trying to **e** establish a secure connection to the server",

**Pull request merged**